src/sentry/db/models/manager.py

```diff
@@ -263,7 +264,7 @@ def contribute_to_class(self, model, name):
         super(BaseManager, self).contribute_to_class(model, name)
         class_prepared.connect(self.__class_prepared, sender=model)
 
-    def get_from_cache(self, **kwargs):
+    def get_from_cache(self, **kwargs) -> t.Any:  # TODO(typing): Properly type this
```

Comment:
We might be able to hook into whatever `django-stubs` is doing to get this to work.

Reply:
Could also at least return that this will be a model